packages/docs/docs/concepts/cornerstone-tools/touch.md

```diff
@@ -0,0 +1,156 @@
+---
+id: touchEvents
+title: TouchEvents
+---
+
+# Touch Events
+
+Touch events are fired when the user touches device with one or more touch points such as a finger or stylus. The flow of touch points are the following:
+
+1. `TOUCH_START`
+2. `TOUCH_START_ACTIVATE`
+3. optional: `TOUCH_PRESS`
+4. optional: `TOUCH_DRAG`
+5. optional: `TOUCH_SWIPE`
+6. `TOUCH_END`
+
+Every time a user places a finger down and lifts it up, the touch order flow will always follow the above. Touch events are mutually exclusive from click events.
+
+Other touch events that can occur are the `TOUCH_TAP` event. A `TOUCH_TAP` will not trigger a `TOUCH_START` event flow. If the user taps successively, only one `TOUCH_TAP` event will fire with the count of how many times the user tapped.
+
+| EVENT                  | Description                                                                                                                                                                                      |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `TOUCH_START`          | Triggers if the user places their touchpoint down for > 50ms.                                                                                                                                    |
+| `TOUCH_START_ACTIVATE` | Triggers only if no tools decided to stop propagation from the `TOUCH_START` event. It is useful to differentiate between touching an existing annotaton, vs needing to create a new annotaiton. |
+| `TOUCH_PRESS`          | Triggers if the user places their touchpoint down and does not move it for > 700ms                                                                                                               |
```

Comment:
same for 700 ms

Reply:
I believe it was what was used in v2 hammerJS also configurable